MovingBlocks/Terasology
# fix: minor fixes for langs and DAG

### Contains
* ru lang correcting
* ua lang - remove duplicating entry.
* DAG - gradle dep fix and change "No rendering module" error message 

### How to test

* Start game without rendering module
* check error message, it is must contains "CoreRendering".
